app/views/simplified-account/settings/email-notifications/collect-email-page.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Settings - Email notifications - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
+{% endblock %}
+
+{% block settingsContent %}
+
+    {{ govukBackLink({
+      text: "Back",
+      href: backLink
+    }) }}
+
+    <form id="collection-email-form" method="post" novalidate>
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+
+      {{
+        govukRadios({
+          idPrefix: 'email-collection-mode',
+          name: 'email-collection-mode',
```

Comment:
use camelCase for names

Reply:
`name` attributes are specifically referenced in controllers when accessing values on the request body and allows for dot notation in the node code, it's also how the design system example components are provided